src/CSSList/CSSList.php

```diff
@@ -413,7 +413,7 @@ protected function renderListContents(OutputFormat $oOutputFormat)
             $oNextLevel = $oOutputFormat->nextLevel();
         }
         foreach ($this->aContents as $oContent) {
-            $sRendered = $oOutputFormat->safely(function () use ($oNextLevel, $oContent) {
+            $sRendered = $oOutputFormat->safely(function () use ($oNextLevel, $oContent): string {
```

Comment:
Could this anonymous function also be made `static`?

Reply:
It can indeed. (I first thought that the `use` made `static` impossible, but was mistaken. Fixed.